README.md

```diff
@@ -140,6 +144,7 @@ To override the **sourceDirectory**, **outputDirectory** and recurse over **sour
             <sourceDirectory>src/main/resources/avro</sourceDirectory>
             <outputDirectory>target/generated-sources</outputDirectory>
             <recursive>true</recursive>
+            <restrictedFields>true</restrictedFields>>
```

Comment:
`restrictedFields` sounds a bit vague to me. Could you make it more explicit by renaming to `limitedNumberOfFieldsInCaseClasses`?